bakery/src/scripts/link_single.py

```diff
@@ -95,13 +95,14 @@ def _parse_tree_for_slug(tree, page_uuid):
 
 
 def patch_link(node, source_book_uuid, canonical_book_uuid,
-               canonical_book_slug, page_slug):
+               canonical_book_slug, page_slug, target_book):
```

Comment:
It looks like you're adding this new `target_book` parameter, but it's never used. Either there is intended logic missing, or the parameter can be removed 🤷‍♂️ ?